sshfs/spec.py

```diff
@@ -147,6 +147,13 @@ async def _info(self, path, **kwargs):
         info["name"] = path
         return info
 
+    @wrap_exceptions
+    async def _modified(self, path: str, **kwargs):
```

Comment:
minor: can we add the return type here?

Reply:
Thanks - added in! Just to check, I don't think anywhere else in the library uses types (I'd instinctively added the hint for path without thinking) are you ok with this PR inadvertantly adding gradual typing?